portfolio/src/main/webapp/index.html

```diff
@@ -11,47 +11,47 @@
 	<a class="active" href="#home">Home</a>
 	<a href="https://www.linkedin.com/in/jonathan-duque-b4290b184/">LinkedIn</a>
 	<a href="https://github.com/duquej">Github</a>
-
-	
 	</div>
   
 	<div class="wrapper">
-		<header>
-			<h1>Jonathan Duque's Portfolio</h1>
-			<p>I'm a sophomore studying Computer Science at Cornell University.
-			My interests are in User Experience/User Design, Programming, and Web Design. 
-			Oh yeah, I'm also a big fan of horticulture!
-			This is one of my many plants, Eric:
-			</p>
-			<img src="images/IMG_1502.JPG" alt="project img">
+	  <header>
+	    <h1>Jonathan Duque's Portfolio</h1>
+		<p>I'm a sophomore studying Computer Science at Cornell University. My interests are 
```

Comment:
You've still got some stray tabs, which might be what's messing up your indentation. Technically the style guide says to use spaces, but I don't actually have a strong opinion on this so I'm happy to leave it up to you and your preferences.